docs/src/Popover.doc.js

```diff
@@ -212,6 +212,33 @@ card(
   />,
 );
 
+card(
+  <MainSection name="Usage guidelines">
+    <MainSection.Subsection columns={2}>
+      <MainSection.Card
+        cardSize="md"
+        type="do"
+        title="When to Use"
+        description={`
+          - Providing additional information for related context without cluttering the surface of a workflow.
+          - Bringing attention to specific user interface elements. In this case, likely used with a [Pulsar](/Pulsar).
```

Comment:
Sre this the so calles educational tooltips? I'd maybe make that more explicit. Annie said that educational are basically the most common case for popovers, Dropdown should be use instead of Popove except if Dropdown doesn't provide enough flexibility.